store/storetest/post_store.go

```diff
@@ -1550,7 +1552,8 @@ func testPostStoreGetFlaggedPosts(t *testing.T, ss store.Store) {
 	err := ss.Preference().Save(&preferences)
```

Comment:
Merging `master` has caused the build to fail. You can fix this by changing this:
```suggestion
	err = ss.Preference().Save(&preferences)
```